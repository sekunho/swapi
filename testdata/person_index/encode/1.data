{"next":"https://swapi.dev/api/people/?format=json&page=2","results":[{"mass":"77.0","height":"172","species":[],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/6/"],"vehicles":["https://swapi.dev/api/vehicles/14/","https://swapi.dev/api/vehicles/30/"],"url":"https://swapi.dev/api/people/1/","eye_color":["blue"],"gender":"male","name":"Luke Skywalker","starships":["https://swapi.dev/api/starships/12/","https://swapi.dev/api/starships/22/"],"edited":"2014-12-20T21:17:56.891Z","skin_color":"fair","created":"2014-12-09T13:50:51.644Z","hair_color":"blond","birth_year":"19.0BBY","homeworld":"https://swapi.dev/api/planets/1/"},{"mass":"75.0","height":"167","species":["https://swapi.dev/api/species/2/"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"vehicles":[],"url":"https://swapi.dev/api/people/2/","eye_color":["yellow"],"gender":"n/a","name":"C-3PO","starships":[],"edited":"2014-12-20T21:17:50.309Z","skin_color":"gold","created":"2014-12-10T15:10:51.357Z","hair_color":"none","birth_year":"112.0BBY","homeworld":"https://swapi.dev/api/planets/1/"},{"mass":"32.0","height":"96","species":["https://swapi.dev/api/species/2/"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"vehicles":[],"url":"https://swapi.dev/api/people/3/","eye_color":["red"],"gender":"n/a","name":"R2-D2","starships":[],"edited":"2014-12-20T21:17:50.311Z","skin_color":"white, blue","created":"2014-12-10T15:11:50.376Z","hair_color":"none","birth_year":"33.0BBY","homeworld":"https://swapi.dev/api/planets/8/"},{"mass":"136.0","height":"202","species":[],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/6/"],"vehicles":[],"url":"https://swapi.dev/api/people/4/","eye_color":["yellow"],"gender":"male","name":"Darth Vader","starships":["https://swapi.dev/api/starships/13/"],"edited":"2014-12-20T21:17:50.313Z","skin_color":"white","created":"2014-12-10T15:18:20.704Z","hair_color":"none","birth_year":"41.9BBY","homeworld":"https://swapi.dev/api/planets/1/"},{"mass":"49.0","height":"150","species":[],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/6/"],"vehicles":["https://swapi.dev/api/vehicles/30/"],"url":"https://swapi.dev/api/people/5/","eye_color":["brown"],"gender":"female","name":"Leia Organa","starships":[],"edited":"2014-12-20T21:17:50.315Z","skin_color":"light","created":"2014-12-10T15:20:09.791Z","hair_color":"brown","birth_year":"19.0BBY","homeworld":"https://swapi.dev/api/planets/2/"},{"mass":"120.0","height":"178","species":[],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"vehicles":[],"url":"https://swapi.dev/api/people/6/","eye_color":["blue"],"gender":"male","name":"Owen Lars","starships":[],"edited":"2014-12-20T21:17:50.317Z","skin_color":"light","created":"2014-12-10T15:52:14.024Z","hair_color":"brown, grey","birth_year":"52.0BBY","homeworld":"https://swapi.dev/api/planets/1/"},{"mass":"75.0","height":"165","species":[],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"vehicles":[],"url":"https://swapi.dev/api/people/7/","eye_color":["blue"],"gender":"female","name":"Beru Whitesun lars","starships":[],"edited":"2014-12-20T21:17:50.319Z","skin_color":"light","created":"2014-12-10T15:53:41.121Z","hair_color":"brown","birth_year":"47.0BBY","homeworld":"https://swapi.dev/api/planets/1/"},{"mass":"32.0","height":"97","species":["https://swapi.dev/api/species/2/"],"films":["https://swapi.dev/api/films/1/"],"vehicles":[],"url":"https://swapi.dev/api/people/8/","eye_color":["red"],"gender":"n/a","name":"R5-D4","starships":[],"edited":"2014-12-20T21:17:50.321Z","skin_color":"white, red","created":"2014-12-10T15:57:50.959Z","hair_color":"none","birth_year":"unknown","homeworld":"https://swapi.dev/api/planets/1/"},{"mass":"84.0","height":"183","species":[],"films":["https://swapi.dev/api/films/1/"],"vehicles":[],"url":"https://swapi.dev/api/people/9/","eye_color":["brown"],"gender":"male","name":"Biggs Darklighter","starships":["https://swapi.dev/api/starships/12/"],"edited":"2014-12-20T21:17:50.323Z","skin_color":"light","created":"2014-12-10T15:59:50.509Z","hair_color":"black","birth_year":"24.0BBY","homeworld":"https://swapi.dev/api/planets/1/"},{"mass":"77.0","height":"182","species":[],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"vehicles":["https://swapi.dev/api/vehicles/38/"],"url":"https://swapi.dev/api/people/10/","eye_color":["blue-grey"],"gender":"male","name":"Obi-Wan Kenobi","starships":["https://swapi.dev/api/starships/48/","https://swapi.dev/api/starships/59/","https://swapi.dev/api/starships/64/","https://swapi.dev/api/starships/65/","https://swapi.dev/api/starships/74/"],"edited":"2014-12-20T21:17:50.325Z","skin_color":"fair","created":"2014-12-10T16:16:29.192Z","hair_color":"auburn, white","birth_year":"57.0BBY","homeworld":"https://swapi.dev/api/planets/20/"}],"previous":"null","count":82}