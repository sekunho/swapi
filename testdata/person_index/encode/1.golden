{"count":82,"next":"https://swapi.dev/api/people/?format=json&page=2","previous":"null","results":[{"birth_year":"19.0BBY","created":"2014-12-09T13:50:51.644Z","edited":"2014-12-20T21:17:56.891Z","eye_color":["blue"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/6/"],"gender":"male","hair_color":"blond","height":"172","homeworld":"https://swapi.dev/api/planets/1/","mass":"77.0","name":"Luke Skywalker","skin_color":"fair","species":[],"starships":["https://swapi.dev/api/starships/12/","https://swapi.dev/api/starships/22/"],"url":"https://swapi.dev/api/people/1/","vehicles":["https://swapi.dev/api/vehicles/14/","https://swapi.dev/api/vehicles/30/"]},{"birth_year":"112.0BBY","created":"2014-12-10T15:10:51.357Z","edited":"2014-12-20T21:17:50.309Z","eye_color":["yellow"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"gender":"n/a","hair_color":"none","height":"167","homeworld":"https://swapi.dev/api/planets/1/","mass":"75.0","name":"C-3PO","skin_color":"gold","species":["https://swapi.dev/api/species/2/"],"starships":[],"url":"https://swapi.dev/api/people/2/","vehicles":[]},{"birth_year":"33.0BBY","created":"2014-12-10T15:11:50.376Z","edited":"2014-12-20T21:17:50.311Z","eye_color":["red"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"gender":"n/a","hair_color":"none","height":"96","homeworld":"https://swapi.dev/api/planets/8/","mass":"32.0","name":"R2-D2","skin_color":"white, blue","species":["https://swapi.dev/api/species/2/"],"starships":[],"url":"https://swapi.dev/api/people/3/","vehicles":[]},{"birth_year":"41.9BBY","created":"2014-12-10T15:18:20.704Z","edited":"2014-12-20T21:17:50.313Z","eye_color":["yellow"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/6/"],"gender":"male","hair_color":"none","height":"202","homeworld":"https://swapi.dev/api/planets/1/","mass":"136.0","name":"Darth Vader","skin_color":"white","species":[],"starships":["https://swapi.dev/api/starships/13/"],"url":"https://swapi.dev/api/people/4/","vehicles":[]},{"birth_year":"19.0BBY","created":"2014-12-10T15:20:09.791Z","edited":"2014-12-20T21:17:50.315Z","eye_color":["brown"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/6/"],"gender":"female","hair_color":"brown","height":"150","homeworld":"https://swapi.dev/api/planets/2/","mass":"49.0","name":"Leia Organa","skin_color":"light","species":[],"starships":[],"url":"https://swapi.dev/api/people/5/","vehicles":["https://swapi.dev/api/vehicles/30/"]},{"birth_year":"52.0BBY","created":"2014-12-10T15:52:14.024Z","edited":"2014-12-20T21:17:50.317Z","eye_color":["blue"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"gender":"male","hair_color":"brown, grey","height":"178","homeworld":"https://swapi.dev/api/planets/1/","mass":"120.0","name":"Owen Lars","skin_color":"light","species":[],"starships":[],"url":"https://swapi.dev/api/people/6/","vehicles":[]},{"birth_year":"47.0BBY","created":"2014-12-10T15:53:41.121Z","edited":"2014-12-20T21:17:50.319Z","eye_color":["blue"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"gender":"female","hair_color":"brown","height":"165","homeworld":"https://swapi.dev/api/planets/1/","mass":"75.0","name":"Beru Whitesun lars","skin_color":"light","species":[],"starships":[],"url":"https://swapi.dev/api/people/7/","vehicles":[]},{"birth_year":"unknown","created":"2014-12-10T15:57:50.959Z","edited":"2014-12-20T21:17:50.321Z","eye_color":["red"],"films":["https://swapi.dev/api/films/1/"],"gender":"n/a","hair_color":"none","height":"97","homeworld":"https://swapi.dev/api/planets/1/","mass":"32.0","name":"R5-D4","skin_color":"white, red","species":["https://swapi.dev/api/species/2/"],"starships":[],"url":"https://swapi.dev/api/people/8/","vehicles":[]},{"birth_year":"24.0BBY","created":"2014-12-10T15:59:50.509Z","edited":"2014-12-20T21:17:50.323Z","eye_color":["brown"],"films":["https://swapi.dev/api/films/1/"],"gender":"male","hair_color":"black","height":"183","homeworld":"https://swapi.dev/api/planets/1/","mass":"84.0","name":"Biggs Darklighter","skin_color":"light","species":[],"starships":["https://swapi.dev/api/starships/12/"],"url":"https://swapi.dev/api/people/9/","vehicles":[]},{"birth_year":"57.0BBY","created":"2014-12-10T16:16:29.192Z","edited":"2014-12-20T21:17:50.325Z","eye_color":["blue-grey"],"films":["https://swapi.dev/api/films/1/","https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/","https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"gender":"male","hair_color":"auburn, white","height":"182","homeworld":"https://swapi.dev/api/planets/20/","mass":"77.0","name":"Obi-Wan Kenobi","skin_color":"fair","species":[],"starships":["https://swapi.dev/api/starships/48/","https://swapi.dev/api/starships/59/","https://swapi.dev/api/starships/64/","https://swapi.dev/api/starships/65/","https://swapi.dev/api/starships/74/"],"url":"https://swapi.dev/api/people/10/","vehicles":["https://swapi.dev/api/vehicles/38/"]}]}